cobalt/build/android/package.json

```diff
@@ -11,8 +11,260 @@
                 "lib.unstripped/libchrobalt.so",
                 "libchrobalt.so",
                 "obj/cobalt/shell/android/cobalt_shell_java_resources.resources.zip",
+                "obj/base/activity_state_java.javac.jar",
```

Comment:
This looks like a lot of maintenance to me. How often does this list change?

Can you create a bug to set up a discussion about alternate solutions?